### ISSUE TITLE
Cameras not working

### ISSUE THREAD

**Issue by jzalesne** [timcharper/homebridge-vivint#43]
I just bought a Hoobs box and installed your vivint plug-in.  Thank you for creating it!!  At first glance, it looks like the sensors are working, however none of the cameras are working.  All the cameras I have are the newest Vivint cameras.  I think they refer to them as Gen 2 cameras.  Any thoughts on why the cameras are not responding?
[Homebridge log](https://github.com/timcharper/homebridge-vivint/files/9977307/Homebridge.log)

**Comment by iNemesis14** [comment]
Did you ever got the cameras to work? Got the same problem

**Comment by jzalesne** [comment]
No---------------------------------------------------------------------------------------------------------------------------Colorado Center for Couples and FamiliesSchedule a session here: https://coloradocouples.com/schedule-now/ Jonathan Zalesne, MA, LPCICEEFT Certified Emotionally Focused Couples Therapist and SupervisorPhone: 303-682-6900 ***@***.***: www.ColoradoCouples.com SOUTH DENVER OFFICE:3600 S. Yosemite Street, Suite 1050, Denver, CO 80237 WHEAT RIDGE OFFICE:7651 W. 41st Ave, Wheat Ridge, CO 80033 FORT COLLINS OFFICE:1502 South College Avenue, Fort Collins, CO  80524 On Dec 19, 2022, at 1:31 AM, iNemesis14 ***@***.***> wrote:﻿
Did you ever got the cameras to work? Got the same problem

**Comment by iNemesis14** [comment]
Managed to get anything else working for vivint? Having such a hard time.
